src/components/new-request/search.vue

```diff
@@ -379,15 +401,24 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
       return !this.showActionNowButton
     }
 
+    // Conditionals for Name Change Flow.
     if (this.isChangeName &&
       this.isNumberedCompany &&
       !this.isSupportedChangeName(this.getEntityTypeCd)
     ) return true
 
     if (this.isChangeNameXpro &&
+      this.isChangeName &&
```

Comment:
I added this here because it was returning true if I was in an XPRO flow in restorations.

Reply:
I was wondering about that.

I'd put isChangeName first so it looks similar to the check above.

Reply:
Thanks. Looks good.